docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -292,6 +292,49 @@ The individual match elements are: -
 * <match-domain-name name="..."/>:: Matches where the protocol of the URI is 'domain' and the scheme specific part of the URI is the name specified within this match element.
  <match-abstract-type name="..." authority="..." />:: Matches the abstract type and/or authority against the values specified within this match element.
 
+=== Elytron client default SSL context security provider
+
+Elytron provides a java security provider which can be used to register a JVM wide default SSLContext. The provider can instantiate SSLContext from Elytron client configuration file which will be returned from the `SSLContext.getDefault()` method. When this provider is registered then all client libraries that use `SSLContext.getDefault()` will use Elytron client configuration without having to add Elytron client API to their code.
```

Comment:
s/default SSLContext/default `SSLContext`

Reply:
s/The provider can instantiate SSLContext from Elytron client configuration file which will be returned from the `SSLContext.getDefault()` method./The provider can instantiate an `SSLContext` from an Elytron client configuration file. This `SSLContext` will then be the one returned when `SSLContext.getDefault()` is called.

Reply:
s/will use Elytron client/will use the Elytron client

Reply:
s/without having to add Elytron client API to their code/without having to use Elytron client APIs in their code

Reply:
Fixed all above. Thanks!